docs/.vitepress/config/en.ts

```diff
@@ -255,6 +255,10 @@ function sidebarDeveloperDocs(): DefaultTheme.SidebarItem[] {
           text: "Junior Developer Program",
           link: "/developer-docs/jr-dev-rules",
         },
+        {
+          text: "Emulator Setup for Developers",
+          link: "/developer-docs/emulator_setup_for_developers",
```

Comment:
I think this needs to be separated by dashes, otherwise it produces a broken link.